theme/src/components/seo.js

```diff
@@ -21,11 +26,12 @@ const SEO = ({ title: pageTitle, description, image: imageURL, article }) => {
   const titleTemplate = getTitleTemplate(metadata)
   const twitterUsername = getTwitterUsername(metadata)
 
-  const title = pageTitle || siteTitle
+  const title = titleTemplate.replace(/%s/g, pageTitle) || siteTitle
```

Comment:
This is what react-helmet does in https://github.com/nfl/react-helmet/blob/b79d30ff44c7ff3c175c31059076e5e549a1b402/src/HelmetUtils.js